### PR TITLE
fixed issue with wrong size of X,Y,Z ranges and added testing

### PR DESCRIPTION
**Pull request by SzymonBlazucki**
As discussed I has changed X,Y,Z to range from 0 to n_voxels - 1 and added test case based on the file:
https://github.com/qiaojunfeng/WannierDatasets/blob/main/datasets/Si2_valence/reference/Si2_valence_00001.cube

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -13,6 +13,7 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 TOML = "fa267f1f-6049-4f14-aa54-33bafae1ed76"
+TestItemRunner = "f8b46487-2199-4994-9208-9a1283c18c0a"
```
Comment (qiaojunfeng) [inline Project.toml]: no need to add dependency here, it is already in the `test/Project.toml`

File: test/volume/cube.jl [hunk 2]
```diff
@@ -0,0 +1,28 @@
+@testitem "read/write cube" begin
+    using LazyArtifacts
+    const Bohr::Float64 = 0.52917721092
```
Comment (qiaojunfeng) [inline test/volume/cube.jl]: you can use this line instead of defining a const here
```
using WannierIO: Bohr
```

File: test/volume/cube.jl [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    cube = read_cube(artifact"Si2_valence/reference/Si2_valence_00001.cube")
+    @test cube.origin ≈ [-7.10461, -9.47281, -9.47281]*Bohr 
+    @test cube.voxel_vectors ≈
+        transpose([0.0 0.39470 0.39470; 0.39470 0.00000 0.39470; 0.39470 0.39470 0.00000])*Bohr 
+    @test cube.X ≈ range(0, 19, 20)
+    @test cube.Y ≈ cube.X
+    @test cube.Z ≈ cube.X
+    @test size(cube.W) == (20,20,20)
+    @test size(cube.atom_positions) == (3,8)
+    print(cube.atom_positions[:,1])
```
Comment (qiaojunfeng) [inline test/volume/cube.jl]: the print line can be removed

**Review by qiaojunfeng — CHANGES_REQUESTED** [review]
Thanks! Just a few changes then we are good to go 🚀

**Review by qiaojunfeng — APPROVED** [review]

File: test/Project.toml [hunk 4]
```diff
@@ -4,7 +4,6 @@ LazyArtifacts = "4af54fe1-eca0-43a8-85a7-787d91b784e3"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-TestItemRunner = "f8b46487-2199-4994-9208-9a1283c18c0a"
```
Comment (qiaojunfeng) [inline test/Project.toml]: ah, should delete the `TestItemRunner` in the `Project.toml` in the root folder, this one in the `test/Project.toml` should be kept.

**Review by qiaojunfeng — CHANGES_REQUESTED** [review]

File: test/Project.toml (hunk 4 above)
Comment (SzymonBlazucki) [reply test/Project.toml]: Yes, so sorry, missclick

**Review by qiaojunfeng — APPROVED** [review]

**Pull request merged**